doc/source/conf.py

```diff
@@ -172,9 +172,9 @@ def setup(app):
 # built documents.
 #
 # The short X.Y version.
-version = 'beta'
+version = '0.4'
```

Comment:
Note that this is one more place to update the version, not that crucial however since it can be fixed afterwards.